config.py

```diff
@@ -1,4 +1,5 @@
 
 CONTEXT_ENVS = ["dev", "staging", "prod"]
 COST_CENTER_CONTEXT = "COST_CENTER"
+COST_CENTER_TAG = 'CostCenter'
```

Comment:
I agree that the naming can be better, but I think `COST_CENTER_KEY` and `COST_CENTER_VALUE` are also confusing because we're still going to do a context lookup in `cdk.json` for `COST_CENTER`. How about `COST_CENTER_VALUE_CONTEXT` and `COST_CENTER_TAG_NAME`?